.chloggen/codeboten_opampsupervisor-emit-otlp.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Out of ignorance: Should this be a breaking change, since this is changing the metric format and port that the internal metrics are being exposed at?

Reply:
i didn't think so since the own metrics were being scraped by the prom receiver and emitted via OTLP anyways